elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -14,6 +14,7 @@
 import com.yahoo.elide.contrib.dynamicconfighelpers.model.Table;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
```

Comment:
We probably don't want to leak the dependency on jsonschema outside the validation logic.